doc/content/GettingStarted/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Quick Start"
+draft: false
+weight: 2
```

Comment:
I don't completely get why each of these files has a specific weight. Also, shouldn't we have a consistent ordering of fields in the frontmatter?

Reply:
Weight define the order of the file. (`_index.html` don't need it though)